lib/components/wallet_information.g.dart

```diff
@@ -127,16 +183,19 @@ class WalletInformationState extends State<WalletInformation> {
                 );
               },
               text: 'Receive',
+              semanticLabel: "Recieve ",
               icon: Icons.qr_code,
             ),
             const SizedBox(width: 8),
             const ActionButton(
               text: 'Send',
               icon: Icons.send,
+              semanticLabel: "Send",
             ),
             const SizedBox(width: 8),
             ActionButton(
                 text: 'Buy GNUS',
+                semanticLabel: "Buy Gnus cryptos for your thining fucking",
```

Comment:
Remove curse words...
```suggestion
                semanticLabel: "Buy Gnus",
```